src/test/java/junit/CalculatorSumTest.java

```diff
@@ -0,0 +1,35 @@
+package junit;
+
+import com.epam.tat.module4.Calculator;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import java.util.logging.Logger;
+
+public class CalculatorSumTest {
+
+    private Calculator calculator;
+    private double firstNumber;
+    private double secondNumber;
+    Logger logger = Logger.getGlobal();
+
+    @Before
+    public void initTest() {
+        calculator = new Calculator();
```

Comment:
I saw this line somewhere else. Is it make sense to create something like BaseTest class, where you initialize your Before and After conditions?

Reply:
I totally agree, thank you.
Changes will be seen in the following commit.